### PR TITLE
Add V by example

### PR DESCRIPTION
**Pull request by vbrazo**
Please complete the following checklist if your PR is adding new link to the list:

- [x] I've read [contributing guidelines](https://github.com/MunGell/awesome-for-beginners/blob/master/CONTRIBUTING.md)
- [x] This PR does not introduce duplicates to the list
- [x] The link is added at the bottom of the list category
- [x] Suggested repository is maintained, have active community, is able to mentor new contributors and have issues with the suggested label
- [x] The link I add follows the suggested pattern:

```
- [Repository Name](link-to-repository-label) _(label: beginner-friendly-label-in-the-repository)_ <br> Description
```

Example link formatting:

```
- [awesome-for-beginners](https://github.com/MunGell/awesome-for-beginners/labels/good-first-contribution) _(label: good-first-contribution)_ <br> A list of awesome beginners-friendly projects.
```

### CODE REVIEW

**Comment by Delta456** [comment]
Thanks for adding this!

File: README.md [hunk 1]
```diff
@@ -33,6 +33,7 @@ If you are not a programmer, but would like to contribute, check out the [Awesom
 - [Rust](#rust)
 - [Scala](#scala)
 - [TypeScript](#typescript)
+- [V](#v)
```
Comment (vbrazo) [inline README.md]: @Delta456 I was wondering if we could add the `v-lang` here as well. We'd need to label an issue as `good first issue` and then link here. I think that'd be great :)

Comment (Delta456) [reply README.md]: Yeah sure

Comment (vbrazo) [reply README.md]: Could you label a couple of issues so we could add it?

**Comment by vbrazo** [comment]
I'm still interested in reviewing this.

**Comment by vbrazo** [comment]
?

**Comment by MunGell** [comment]
Hey @vbrazo is the questions for me?

I saw some internal review/communication in this PR, was not sure if it is already ready.

I have reviewed the repository now, but there are no beginner-friendly issues (except an introductory one) in it.

Feel free to re-open PR when you get more beginner-friendly issues.

**Pull request closed without merging**